package.json

```diff
@@ -7,6 +7,7 @@
     "@testing-library/react": "^11.1.0",
     "@testing-library/user-event": "^12.1.10",
     "react": "^17.0.1",
+    "react-bootstrap": "^1.4.3",
```

Comment:
Not really a problem for this assignment cause the goal is basically to get familiar with Git. But in the future, we should either use react-bootstrap or reactstrap

Reply:
Yeah, it would be more organized. It was my misunderstanding; thought we were using react-bootstrap but I'll ask for clarification next time.